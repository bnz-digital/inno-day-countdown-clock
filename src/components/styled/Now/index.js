import React from 'react'
import styled from 'styled-components'

const StyledNow = styled.p`
  font-size: 2vw;
`
StyledNow.displayName = 'StyledNow'

export default function Now ({ children }) {
  return (
    <StyledNow>
      Now:
      {' '}
      {children}
    </StyledNow>
  )
}
