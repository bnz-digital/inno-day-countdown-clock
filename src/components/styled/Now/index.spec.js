import React from 'react'
import { shallow } from 'enzyme'

import Now from '.'

describe('components:styled:Now', () => {
  it('renders <Now/> correctly', () => {
    expect(toJson(shallow(<Now />))).toMatchSnapshot()
  })
})
