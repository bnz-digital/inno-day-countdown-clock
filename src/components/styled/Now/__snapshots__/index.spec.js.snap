// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:Now renders <Now/> correctly 1`] = `
<StyledNow>
  Now:
   
</StyledNow>
`;
